src/components/grid-list/grid-list.scss

```diff
@@ -1,4 +1,13 @@
-// TODO(kara): Review this to see if MD spec updates are needed
+/* height of tile header or footer if it has one line */
+$md-grid-one-line-height: 48px;
+/* height of tile header or footer if it has two lines */
+$md-grid-two-line-height: 68px;
+/* side padding for text in tile headers and footers */
+$md-grid-text-padding: 16px;
+
+/* font styles for text in tile headers and footers */
+$md-grid-font-size: 16px;
+$md-grid-secondary-font: 12px;
```

Comment:
Can the variables be `gridlist` instead of `grid`? Many people refer to 'data-table" as "grid", so I want to avoid confusion.